measure_extinction/extdata.py

```diff
@@ -551,10 +551,16 @@ def trans_elv_alav(self, av=None, akav=0.112):
             if av is None:
                 if "AV" not in self.columns.keys():
                     self.calc_AV(akav=akav)
-                av = _get_column_val(self.columns["AV"])
+            fullav = self.columns["AV"]
+            if len(np.atleast_1d(fullav)) == 1:
+                fullav = np.array([fullav, 0.0])
             for curname in self.exts.keys():
-                self.exts[curname] = (self.exts[curname] / av) + 1
-                self.uncs[curname] /= av
+                tuncs = np.sqrt(
+                    np.square(self.uncs[curname] / self.exts[curname])
+                    + np.square(fullav[1] / fullav[0])
```

Comment:
If the uncertainties are asymmetric (i.e. A(V) has 2 uncertainty values), this will technically only take the "minus" uncertainty.

Reply:
Good point.  Not quite sure the best solution at this point.  Maybe just average the punc and munc for the av_unc?

Reply:
I think in this case, taking the average is ok. I have 2 uncertainties on A(V) because of the emcee fitting, but they are not very different. Is this something that should be built into this function, or should we make sure the A(V) that goes into this function already has the right format?

Reply:
Fixed.